data/com.endlessm.GoogleChromeInitialSetup.gschema.xml

```diff
@@ -3,8 +3,8 @@
     <schema path="/com/endlessm/GoogleChromeInitialSetup/" id="com.endlessm.GoogleChromeInitialSetup">
         <key type="b" name="enabled">
             <default>false</default>
-            <summary>Enabled</summary>
-            <description>Enable Google Chrome installation</description>
+            <summary>Install Google Chrome on start up</summary>
+            <description>When enabled installs Google Chrome on start up</description>
```

Comment:
Even if this is a step forward already, I think an even more correct description would be "When enabled, Google Chrome is automatically installed after logging in for the first time"